ocs_ci/utility/utils.py

```diff
@@ -1303,3 +1304,39 @@ def wait_for_co(operator):
     from ocs_ci.ocs.ocp import OCP
     ocp = OCP(kind='ClusterOperator')
     ocp.get(operator)
+
+
+def censore_values(data_to_censore, keys):
+    """
+    This function censores values in dictionary for passed keys
+
+    Args:
+        data_to_censore (dict): data to censore
+        keys (list): keys where to censore the values in data dictionary
+
+    """
+    for key in keys:
+        if key in data_to_censore:
+            data_to_censore[key] = '*****'
+
+
+def print_cofig_to_file(file_path):
```

Comment:
*config

Reply:
Also, "print" doesn't seem like the correct word to use here. "write"?

Reply:
> Also, "print" doesn't seem like the correct word to use here. "write"?

What about "dump" instead of print or write? (just suggestion)